include/fastdds/dds/publisher/DataWriter.hpp

```diff
@@ -90,9 +90,7 @@ class DataWriter : public DomainEntity
     /**
      * Write data to the topic.
      * @param data Pointer to the data
-     * @return True if correct
-     * @par Calling example:
-     * @snippet fastrtps_example.cpp ex_PublisherWrite
+     * @return True if correct, false otherwise
      */
     RTPS_DllAPI bool write(
             void* data);
```

Comment:
Should we change the return type of this to `ReturnCode_t` (perhaps on a different PR)?

Reply:
It is a non-standard method, but if you want to change the return type I think it is better to do it in a separate PR